cryptography/src/commonJvmAndroid/kotlin/com.wire.kalium.cryptography/MLSClientImpl.kt

```diff
@@ -0,0 +1,103 @@
+package com.wire.kalium.cryptography
+
+import com.wire.crypto.CiphersuiteName
+import com.wire.crypto.ConversationConfiguration
+import com.wire.crypto.CoreCrypto
+import com.wire.crypto.Invitee
+import java.time.Duration
+
+@OptIn(ExperimentalUnsignedTypes::class)
+actual class MLSClientImpl actual constructor(
+    rootDir: String,
+    databaseKey: String,
+    clientId: CryptoQualifiedClientId) : MLSClient {
+
+    private var coreCrypto: CoreCrypto
```

Comment:
Suggestion to make coreCrypto immutable 
```suggestion
    private val coreCrypto: CoreCrypto
```